python/taichi/lang/matrix.py

```diff
@@ -914,7 +914,7 @@ def field(cls,
               name="",
               offset=None,
               needs_grad=False,
-              layout=None):  # TODO(archibate): deprecate layout
+              layout=Layout.AOS):  # TODO(archibate): deprecate layout
```

Comment:
is this TODO still valid after we change it to Layout enum?

Reply:
I don't know what was planned in the long term from this TODO..

Reply:
Let's remove that. We do want to support AOS/SOA layout

Reply:
done